Rewrite docs for core concepts (#395)
kitbagjs/router: docs/core-concepts/router-route.md

# Router Route

> The current route is represented by the "Router Route". See the [RouterRoute](/api/types/RouterRoute.md) type for more information.

Reviewer: ```suggestion
The current route is represented by the [RouterRoute](/api/types/RouterRoute.md).
```

> The route can be accessed using the [useRoute](/composables/useRoute.md) composable within your components. It is also available on the router instance as the `route` property.

Reviewer: ```suggestion
This current route is accessed using the [useRoute](/composables/useRoute.md) composable within your components. It is also available on the router instance as the `route` property.
```
